data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
+				to display
+					"reputation: Hai (Unfettered)" >= 30
+			`Choot'k greets you, with a worried-looking Batu at his side. "Welcome back, Captain <last>! We found many spoils of war on this planet. The pirates' bunkers had plenty of gadgetry littering their bunkers, including limited-access human goods and Hai equipment. We've found a few Korath outfits, too, including some quite advanced ones. It looks like these pirates must have gotten lucky and managed to actually loot one of their ships! Navy inventory crewmen are gathering it to secure areas for inventory and analysis before divvying it up."`
+				to display
+					"reputation: Hai (Unfettered)" < 30
+			`	Batu looks less satisfied. "We've found only a few dozen items, and there aren't any live Korath - or even Korath bodies. With all the fighters being disabled, we'd expect to find a bit more. This is likely part of an operation to pillage and sell Korath outfits. Worse, they may even be trading with the Korath. The operation's Stormhold facilities were for showcasing and auctioning Korath goods. By taking their sales center, we may have shut down the operation."`
+
+			label questions
+			choice
+				`	"Who are the Korath?"`
+					to display
+						not "know who the korath are"
+						"reputation: Korath" > -100
+						not "asked choot'k about the korath"
+				`	"I've fought the Korath for a long time, but I don't really know who they are."`
+					to display
+						not "know who the korath are"
+						"reputation: Korath" <= -100
+						not "asked choot'k about the korath"
+				`	"What can you tell me about the Korath?"`
+					to display
+						not "asked choot'k about the korath"
+						or
+							has "know who the korath are"
+							"reputation: Korath" <= -100
+				`	"Why are there so many Korath artifacts?"`
+					goto somany
+			action
+				set "know who the korath are"
+				set "asked choot'k about the korath"
+			`	Choot'k responds. "The Korath are a group of lizards that lived south of Hai space until a few centuries ago. They used to be obsessed with art, music, and other esoterics. I care more for practical matters, but I must admit that I enjoy our old recordings of their music. Haunting guttural hums accented by angelic vocals with lyrics in their intricate language. The mood goes from eerie and hopeful to enlivening to frightening from verse to verse and draws the listener with them. Accompanied by mesmerizing visuals, the experience absorbs you in alluring totality."`
```

Comment:
Would Choot'k call them lizards, or a lizard-like species? Is it meant to sound like he doesn't like them?

Reply:
Referring back to the [Korath lore document](https://docs.google.com/document/d/1875bt5lLh4GH_TE_9EsoAvwwk0DPJnLkVt-ztTvfv1g/edit#), they are a "species of partially-feathered, hyper-adaptable, omnivorous *lizard*." He is merely stating a fact. It's like saying that humans are hairless bipeds. To someone who has never seen a human, it is an invaluable piece of information.

Reply:
If he had said, "feather-munching, nigh-cannibalistic, apocalypse-obsessed, cockroach-fearing, dancing, art freaks," then *that* would be insulting, albeit essentially true.

Reply:
well in a way calling them lizards is like calling us apes and I would prefer if he didnt have a grudge against them (for the Unfettered campaign)
so good you specified that

Reply:
No, there would be no grudge. The Hai and Korath were friends for two millennia, and the Hai had to watch helplessly as the Korath destroyed themselves. This wasn't ancient history, either, because there are Hai still alive from that time. They would be in mourning, not enraged.

Reply:
Indeed, the views of the people who actually fought the Korath (Unfettered Hai) would be different from the ones watching the suffering from afar (Fettered Hai).